examples/wasm-demo/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "wasm-demo"
+description = "A demo using cairo-vm in a WASM environment"
+version.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+readme.workspace = true
```

Comment:
I see we have a README made specifically for the demo. Are we sure we don't want to point to that one?

Reply:
You're right! I didn't think about that.